src/main/resources/application.properties

```diff
@@ -0,0 +1,16 @@
+#MySQL database credentials
+spring.datasource.driverClassName=com.mysql.cj.jdbc.Driver
+spring.datasource.url=jdbc:mysql://:3306/utopia
+spring.datasource.username=
+spring.datasource.password=
+
+spring.mvc.throw-exception-if-no-handler-found=true
```

Comment:
Comments explaining what each of these properties does and/or why we need it would be very helpful. Or if any of them is not necessary (e.g. if you're setting it to its default value) just remove it.